BUILD.bazel

```diff
@@ -2,8 +2,9 @@ package(default_visibility = ["//visibility:public"])
 
 exports_files([
     "tsconfig.json",
-    "node_modules/zone.js/dist/zone.min.js",
-])
+] + glob([
+    "node_modules/zone.js/dist/*.js",
```

Comment:
Do you need to export more than zone.min.js? I could see you also exporting the non-minified version for some small cases but I don't think you should ever have a reason to refer to any other part of zone.js

Reply:
nope, I noticed that too. I previously had the file list in a different BUILD file which is why I had to export them.
Fixed.